pkgs/development/python-modules/aenum/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "aenum";
+  version = "1.4.7";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
```

Comment:
does fetchPypi take pname? Should this buildPythonPackage call be embedded directly in python-packages.nix?